pages/projects/[id].tsx

```diff
@@ -116,6 +121,46 @@ const Project: NextPage = (): JSX.Element => {
       return;
     }
 
+    const landingIndex =
```

Comment:
I need to extract this whole method out into some kind of saga/action, (i think we need to do it for all), but we'll see.